locales/it/index.yml

```
@@ -350,7 +350,8 @@ wallet:
     add: aggiungi
     addButton: Aggiungi un metodo
     addDescription: Puoi aggiungere i tuoi metodi di pagamento (carte, conti correnti,
-      ecc.) per non doverli inserire ad ogni transazione
+      ecc.) per non doverli inserire ad ogni transazione.
+    walletAlert: "Attenzione: i tuoi metodi di pagamento non sono attualmente supportati"
```

```suggestion
    walletAlert: "Attenzione: nel tuo portafoglio ci sono registrati dei metodi di pagamento ma nessuno di questi è compatibile con l'app"
```
@matteodesanti I would like the user knows we successfully loaded his wallet but actually we don't support any of them. What do you think?

---

I message is detailed as proposed within https://github.com/teamdigitale/io-app/pull/1212/commits/f821fdc496ec92250760972e480ef5c6c44bef8b